src/pages/SongPage.tsx

```diff
@@ -35,6 +48,33 @@ const SongPage: React.FC = () => {
     };
   }, [bookId]);
 
+  useEffect(() => {
+    if (isDesktop()) {
+      return;
+    }
+    const gesture: Gesture = createGesture({
```

Comment:
can we abstract out this logic to some Utils so we also don't have to import the constants here?